internal/v4/common_test.go

```diff
@@ -142,6 +143,7 @@ func (s *commonSuite) startServer(c *gc.C) {
 		StatsCacheMaxAge: time.Nanosecond,
 		MaxMgoSessions:   s.maxMgoSessions,
 	}
+	keyring := bakery.NewPublicKeyRing()
```

Comment:
or just use httpbakery.NewPublicKeyRing - then you won't need to explicitly add the public key below.